### ISSUE TITLE
How Syncronized BatchNorm implemented

### ISSUE THREAD

**Issue by Podidiving** [ternaus/retinaface#8]
Hi! In README you mentioned, that you've implemented Syncronized BatchNorm with pytorch lightning, but I don't see it in code. Could you please point me, where this feature is implemented? Thanks in advance)

**Comment by ternaus** [comment]
https://github.com/ternaus/retinaface/blob/master/retinaface/train.py#L45
